extensions-core/kafka-indexing-service/src/main/java/org/apache/druid/indexing/kafka/LegacyKafkaIndexTaskRunner.java

```diff
@@ -307,8 +307,8 @@ private TaskStatus runInternal(TaskToolbox toolbox) throws Exception
         final SeekableStreamEndSequenceNumbers<Integer, Long> restoredNextPartitions = toolbox.getObjectMapper().convertValue(
             restoredMetadataMap.get(METADATA_NEXT_PARTITIONS),
             toolbox.getObjectMapper().getTypeFactory().constructParametrizedType(
```

Comment:
Curious how this ever worked. It is assigned to a `SeekableStreamEndSequenceNumbers`, but Jackson expects a `SeekableStreamStartSequenceNumbers`.